paasta_tools/kubernetes_tools.py

```diff
@@ -910,10 +910,17 @@ def get_autoscaling_metric_spec(
             max_replicas,
             autoscaling_params,
         )
+
+        labels = {
```

Comment:
might be nice to add the pool here too - then we can easily grab all the HPAs for a given pool